modules/lead/product-aws/aws.tf

```diff
@@ -77,31 +116,51 @@ resource "aws_codebuild_project" "codebuild_staging" {
 resource "aws_codebuild_project" "codebuild_production" {
   for_each = var.pipelines
 
-  name          = "${var.product_name}-${each.value.repo}-production"
+  name          = replace("${var.product_name}-${each.value.repo}-production", "/^${var.product_name}-${var.product_name}/", var.product_name)
   description   = "terraform_codebuild_project"
-  build_timeout = "5"
+  build_timeout = "10"
   service_role  = var.codebuild_role
 
   environment {
-    compute_type                = "BUILD_GENERAL1_SMALL"
-    image                       = "aws/codebuild/amazonlinux2-x86_64-standard:2.0"
+    compute_type                = "BUILD_GENERAL1_LARGE"
+    image                       = "${var.toolchain_image_repo}/builder-image-skaffold:${var.builder_images_version}"
     type                        = "LINUX_CONTAINER"
+    image_pull_credentials_type = "SERVICE_ROLE"
+
+    environment_variable {
+      name  = "PROD_NAMESPACE"
+      value = "${var.product_name}-production"
+    }
+    environment_variable {
+      name  = "ISTIO_DOMAIN"
+      value = "${var.product_name}-production.${var.cluster}.prod.liatr.io"
```

Comment:
should be `-prod` since that is the dns used for the wildcard cert.

Reply:
we should be using `${var.cluster_domain}` instead of `${var.cluster}.prod.liatr.io` - otherwise this won't work in sandbox or any aws account we stand up for a new lab

Reply:
fixed to use `${var.cluster_domain}` and naming for pod